spec/factories/admin_users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :admin_user do
+    email { [Faker::Internet.email].sample }
```

Comment:
The problem with your approach is that email format is validated by Devise. `"#{Faker::Internet.email} #{n}"` would generate e.g. "admin@example.com 1", which would fail the validation. The sequence number must be appended to the front of the address or internally, but without interfering with the email's domain format.